CODE-GUIDELINES.md

```diff
@@ -187,6 +187,22 @@ const text = getByText('text in the page');
 expect(text).toHaveStyle({ color: '#FFFFF'});
 ```
 
+### `waitFor` vs `waitUntil`
```

Comment:
Could you link the vitest documentation

- https://vitest.dev/api/vi.html#vi-waitfor
- https://vitest.dev/api/vi.html#vi-waituntil